doc/reference/configuration/configuration_reference.rst

```diff
@@ -2427,8 +2475,26 @@ The ``process`` defines configuration parameters of the Tarantool process in the
 
 .. confval:: process.title
 
-    Sets the name of the server's process title (what’s shown in the COMMAND column for
-    ``ps -ef`` and ``top -c`` commands).
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Add the given string to the server's process title
+    (e.g. it's shown in the COMMAND column for the Linux commands
+    ``ps -ef`` and ``top -c``).
+
+    For example, if you set ``title: ''``, :samp:`ps -ef` shows
+    the Tarantool server process thus:
+
+    .. code-block:: console
+
+        $ ps -ef | grep tarantool
+        1000     14939 14188  1 10:53 pts/2    00:00:13 tarantool <running>
```

Comment:
I think that an empty value is not the best example to place it first.

How about something like `myservice - {{ instance_name }}`?